elm/web/rhub.py

```diff
@@ -0,0 +1,601 @@
+"""
+Code to build Corpus from the researcher hub.
+"""
+import os
+import os.path
+import logging
+from urllib.request import urlopen
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+from rex import init_logger
+
+logger = logging.getLogger(__name__)
+init_logger(__name__, log_level='DEBUG')
+init_logger('elm', log_level='INFO')
```

Comment:
you should never initialize loggers in module files. Our logging theory is this: every module file includes a logger object (line 13 here) and does a bunch of logging. Anyone building software on our repo or running scripts with our code can then selectively choose which messages to turn on. We should not turn on logging by default for others, that is annoying.